algoliasearch-core/src/main/java/com/algolia/search/models/common/Log.java

```diff
@@ -140,6 +152,8 @@ public String toString() {
         + ", sha1='"
         + sha1
         + '\''
+        + ", innerQuery="
+        + innerQueries
```

Comment:
Just to stay consistent with the rest of the formatting, could you change those two lines with:

```java
        + ", innerQueries='"
        + innerQueries
        + '\''
```

(name changed to `innerQueries` and added single-quote wrapping)

Reply:
Yup nice catch!
Fixed :)